### PR TITLE
Miljo/fixing date issue

### PR DESCRIPTION
**Pull request by battletoilet**
#### Description of changes
Adding the date to the end of the CDN URL so that it updates everyday. 
Adding some more logic to check if the current environment is localhost

#### Focus areas to test
Fabric website localhost.

### CODE REVIEW

File: apps/fabric-website/index.html [hunk 1]
```diff
@@ -79,39 +105,36 @@
...
-        var varCustomerTracking = 1;
-        var varAutoFirePV = 1;
-        var Route = "50000";
-        var Ctrl = "SD300";
-        document.write("<script type='text/javascript' src='" + (window.location.protocol) + "//c.microsoft.com/ms.js'" + "'><\/script>");
-    </script>
-<noscript><img alt="" width="1" height="1" src="http://c.microsoft.com/trans_pixel.aspx" /></noscript>
-<script>
+  </script>
+  <script type="text/JavaScript">
```
Comment (dzearing) [inline apps/fabric-website/index.html]: text/javascript

File: apps/fabric-website/index.html [hunk 2]
```diff
@@ -79,39 +105,36 @@
...
+    var varSegmentation = 0; var varClickTracking = 1; var varCustomerTracking = 1; var varAutoFirePV = 1; var Route = "50000";
+    var Ctrl = "SD300"; var script = document.createElement('script'); script.setAttribute('type', 'text/javascript'); script.src
```
Comment (dzearing) [inline apps/fabric-website/index.html]: this is super weird... why getting rid of CRs?

What is ms.js?

what is trans_pixel.aspx?

Comment (battletoilet) [inline apps/fabric-website/index.html]: @dzearing vscode was doing that automatically and I'm not sure why 😕

**Comment by battletoilet** [comment]
Approved

File: apps/fabric-website/index.html [hunk 3]
```diff
@@ -80,39 +105,35 @@
...
+      i['GoogleAnalyticsObject '] = r; i[r] = i[r] || function () {
         (i[r].q = i[r].q || []).push(arguments)
       }, i[r].l = 1 * new Date(); a = s.createElement(o),
         m = s.getElementsByTagName(o)[0]; a.async = 1; a.src = g; m.parentNode.insertBefore(a, m)
-    })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
+    })(window, document, 'script ', '//www.google-analytics.com/analytics.js ', 'ga ');
 
-    if (window.location.host.indexOf('localhost') != -1 || window.location.host.indexOf('azurewebsites') != -1) {
-      ga('create', 'UA-65962112-1', 'auto'); //staging and Dev GA
+    if (window.location.host.indexOf('localhost ') != -1 || window.location.host.indexOf('azurewebsites ') != -1) {
```
Comment (cschleiden) [inline apps/fabric-website/index.html]: why add all these spaces or is it only the diff? 

image

**Pull request merged**